Source/diablo.h

```diff
@@ -49,7 +49,7 @@ enum class MouseActionType : int {
 	Spell,
 	SpellMonsterTarget,
 	SpellPlayerTarget,
-	Attack,
+	MeleeAttack,

```

Comment:
Was this intended to be part of the MonsterMode changes? Looks like a find-replace casualty

Reply:
This was a mistake. Intended to rename the equivalent entry on `MonsterMode`.

Reply:
Fixed. There were other renames as well that were incorrect.